src/scripts/main.js

```diff
@@ -1,3 +1,54 @@
 'use strict';
 
 // write code here
+
+const employees = document.querySelectorAll('li');
+const listEmployees = document.querySelector('ul');
+
+function getSalary(element) {
+  const salary = element.dataset.salary;
+
+  if (salary) {
+    let cleanedSalary = salary.slice(1);
+
+    cleanedSalary = cleanedSalary.split(',').join('');
+
+    return Number(cleanedSalary);
+  }
+
+  return 0;
+}
+
+function sortList(list) {
+  const listItem = [...list];
+
+  return listItem.sort((a, b) => getSalary(b) - getSalary(a));
```

Comment:
The `sortList` function is currently modifying the original list of employees, which could lead to unexpected side effects. It would be safer to create a new array with the sorted elements instead of sorting the original array in place.